src/main/kotlin/com/jetbrains/snakecharm/inspections/SmkSeveralRulesAreOverriddenAsOneInspection.kt

```diff
@@ -25,7 +27,7 @@ class SmkSeveralRulesAreOverriddenAsOneInspection : SnakemakeInspection() {
 
         override fun visitSmkUse(use: SmkUse) {
             val name = use.nameIdentifier
-            if (name == null || name.text.contains('*')) {
+            if (name == null || name.text.contains('*') || name.elementType == SmkElementTypes.USE_IMPORTED_RULES_NAMES) {
```

Comment:
Could we add some API method in SmkUse. Maybe reuse/refactor `SmkUse.hasPatternInDefinitionOfInheritedRules()` ?